lib/srv/desktop/tdp/proto.go

```diff
@@ -346,6 +349,28 @@ func (r RDPResponsePDU) Encode() ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
+// RDPChannelIDs are the io and user channel ids negotiated during the RDP connection.
```

Comment:
Can we link to some part of the spec that might help explain why this is necessary?